go/vt/vttablet/tabletserver/vstreamer/testenv/testenv.go

```diff
@@ -41,6 +41,8 @@ import (
 	vttestpb "vitess.io/vitess/go/vt/proto/vttest"
 )
 
+const TestDBName = "vttest"
```

Comment:
extremely minor nit but referring to this externally reads as `testenv.TestDBName` which might be a bit clunkier than `testenv.DBName`